lib/nodes/addon/components/driver-azure/template.hbs

```diff
@@ -264,13 +376,35 @@
         <label class="acc-label">
           {{t "nodeDriver.azure.size.label"}}
         </label>
-        {{new-select
-          classNames="form-control"
-          content=sizeChoices
-          optionLabelPath="value"
-          optionGroupPath="group"
-          value=config.size
-        }}
+        {{#if vmSizesLoading}}
+          <i class="icon icon-spinner icon-spin"></i>
+        {{else}}
+          {{new-select
```

Comment:
This can be a really long list, it miiiight be better to use a searchable-select; however, that component is a different height than the other input in this row so it is kinda ugly:
![Screen Shot 2022-12-19 at 2 37 46 PM](https://user-images.githubusercontent.com/42977925/208528959-18decf4d-6395-40d1-9af2-11093ac87900.png)

new-select and searchable-select have different padding and afaict that was accidental. seachable-select was given `padding: 10px` to match new-select back in Jan, then in July new-select was switched from `padding: 10px` to `padding: 4px 10px` to fix a clipping issue when the height was forced to match other inputs (text inputs have `padding: 5px 10px`) those PRs are, respectively, https://github.com/rancher/ui/pull/4805 and https://github.com/rancher/ui/pull/4855

I think the fix would be to change searchable-select to `padding: 4px 10px`. But I don't see any of this as essential to this PR so feel free to ignore the entirety of this comment if you'd prefer not to mess with such a widely-used component.

Reply:
I agree that searchable select would be more friendly, but in my setup it's not just the padding that looks weird, but also the height. The searchable select is so tall that it extends past the form:
<img width="1249" alt="Screenshot 2022-12-19 at 3 57 41 PM" src="https://user-images.githubusercontent.com/20599230/208543949-6c4fb068-6920-476b-99b3-7abc39a034f6.png">

For fear of making breaking changes to the searchable select, I'll leave it as new-select for now, but I'll open a separate issue about fixing the styling for that component.

Reply:
Oof, yeah, I agree about that being too much to tack onto this PR.